owners/index.js

```diff
@@ -46,11 +48,28 @@ module.exports = app => {
     });
   }
 
+  /**
+   * Identify if a PR is in a state that reviewers should be assigned.
+   *
+   * @param {!object} pr GitHub PR payload.
+   * @return {boolean} false if the PR is a draft or the title contains WIP/DNS.
+   */
+  function shouldAssignReviewers(pr) {
+    return !(pr.draft || NO_REQUEST_OWNERS_REGEX.test(pr.title));
```

Comment:
So we're not looking at the description? A lot of people use the description to add WIP

Reply:
There's a much higher chance of a PR description containing something like "ensure we do not submit broken code in case of ..." or other accidental collisions. This isn't supposed to catch everything, just the cases where it's obvious enough that it annoys people.